front/app/containers/Authentication/tracks.ts

```diff
@@ -10,17 +9,8 @@ export default {
     'Sign up - email & password sign-up step entered',
   signUpEmailPasswordStepExited:
     'Sign up - email & password sign-up step exited',
-  signUpEmailPasswordStepCompleted:
-    'Sign - up email & password sign-up step completed',
-  signUpEmailPasswordStepFailed:
-    'Sign up - email & password sign-up step failed',
```

Comment:
A lot of these were already unused, others became irrelevant as the sign up flow merged with the missing data flow. So I just deleted them. Depending on https://go-vocal.slack.com/archives/C65GX921W/p1724266496827569, we can decide to put them back somehow or otherwise delete all of these